Extension/package.json

```diff
@@ -2327,13 +2327,13 @@
           "markdownDescription": "%c_cpp.configuration.simplifyStructuredComments.markdownDescription%",
           "scope": "application"
         },
-        "C_Cpp.autocompleteDoxygenComment":{
+        "C_Cpp.autoCompleteDoxygenComment":{
```

Comment:
I think it's supposed to be "autocomplete". Lower-case c.

Reply:
Sorry for the changes, I didn't know autocomplete is supposed to be one word. If "c" was capitalized, then it will show up as two words on the VS Code UI settings. Other settings that use the word autocomplete currently displays it as one word. So for consistency, this should be `autocompleteDoxygenComment`.